authors/apps/comments/views.py

```diff
@@ -175,3 +176,40 @@ def post(self, request, **kwargs):
             return Response({"message": "Your like has been cancelled"},
                             status=status.HTTP_200_OK)
 
+
+class CommentEditHistoryAPIView(APIView):
+    """
+    GET History/
+    Return comment updates history by a given user
+    on a particular article.
+    """
+
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request, pk, article_id, **kwargs):
+        """
+        Return all comment update history.
+        """
+        get_article(article_id)
+        comment = get_comment(pk)
+        comment_history = comment.history.all()
+        update_history = []
+        for update in list(comment_history):
```

Comment:
@mwinel If we can implement the method suggested above there will be no need for this.